test/cppsuite/test_harness/workload/workload_validation.h

```diff
@@ -47,22 +51,24 @@ class workload_validation {
     /*
      * Validate the on disk data against what has been tracked during the test. This is done by
      * replaying the tracked operations so a representation in memory of the collections is created.
-     * This representation is then compared to what is on disk. operation_table_name: collection
-     * that contains all the operations about the key/value pairs in the different collections used
-     * during the test. schema_table_name: collection that contains all the operations about the
-     * creation or deletion of collections during the test.
+     * This representation is then compared to what is on disk.
+     *
+     * - operation_table_name: Table that contains all the operations performed on keys.
+     * - schema_table_name: Table that contains all the schema operations performed.
      */
     void
     validate(const std::string &operation_table_name, const std::string &schema_table_name,
-      database &database)
+      const std::vector<uint64_t> &known_collection_ids)
     {
         WT_DECL_RET;
-        wt_timestamp_t key_timestamp;
+        wt_timestamp_t tracked_timestamp;
         std::vector<uint64_t> created_collections, deleted_collections;
-        uint64_t key_collection_id;
-        const char *key, *value;
-        int value_operation_type;
-        uint64_t collection_id = UINT64_MAX;
+        uint64_t tracked_collection_id, collection_counter, expected_collection_id;
```

Comment:
Do we want to enforce the alphabetic stuff? If so, then we should reorder them.

Reply:
I'll reorder it.